### DISCUSSION TITLE
Styling based on parent or sibling state doesn't work in nuxt 3

### DISCUSSION THREAD

**Discussion by smartfox87** [tailwindlabs/tailwindcss#11254] [Help] [answered]
Styling based on parent or sibling state doesn't work in nuxt 3
package.json
  "devDependencies": {
    "nuxt": "3.5.0"
  },
  "dependencies": {
    "autoprefixer": "10.4.14",
    "postcss": "8.4.21",
    "tailwindcss": "3.2.7"
  }

[example of error on codesandbox](https://codesandbox.io/p/sandbox/tailwind-nuxt-3-3bb3jy?file=%2Fapp.vue%3A9%2C32)

**Comment by thecrypticace** [comment, marked as answer]
Hey! You need to use the `peer-*` variant to do this. If you want to customize the name to "scope" the variant you can use something like `peer-hover/title:underline` or `peer-hover/input:underline` and the targeted class names would be `peer/title` and `peer/input` respectively.

I've forked and updated your codesandbox showcasing how to do what you're after: https://codesandbox.io/p/sandbox/tailwind-nuxt-3-forked-oidnj7?file=%2Fapp.vue%3A6%2C50

I also left a note in there regarding this:
```
<div class="title-[.underline]:underline">text 1</div>
```

I'm assuming you want this to be underlined if the title is underlined. Such a feature doesn't exist in the browser except perhaps by using container queries but they're not well supported yet.

But I'll reproduce the code here as well:
```html
  <div>
    <h1 class="peer/title text-3xl font-bold underline">Hello world!</h1>
    <div class="peer-[&:hover]/title:underline">text 1</div>
    <div class="peer-hover/title:underline">text 2</div>
    <input type="checkbox" checked class="peer/input" />
    <div class="peer-checked/input:underline">text 3</div>
  </div>
```

Hope that helps!
